packages/x-scheduler/src/joy/event-calendar/EventCalendar.tsx

```diff
@@ -12,9 +12,21 @@ export const EventCalendar = React.forwardRef(function EventCalendar(
   const { events, onEventsChange, className, ...other } = props;
 
   return (
-    <div className="mui light">
-      <div className={clsx(className, 'EventCalendar')} ref={forwardedRef} {...other}>
-        HELLO WORLD ({events.length} events loaded)
+    <div
```

Comment:
I kept everything div/ span but if you have proposals for a more semantic HTML I'm interested  :laughing: